docs/api/model.md

```diff
@@ -265,8 +265,10 @@ Run an aggregation method on the specified field
 | [options] | Object | Query options. See sequelize.query for full options |
 | [options.dataType] | DataType &#124; String | The type of the result. If `field` is a field in this Model, the default will be the type of that field, otherwise defaults to float. |
 | [options.distinct] | boolean | Applies DISTINCT to the field being aggregated over |
+| [options.plain] | boolean | Casts the first returned value of `aggregateFunction` to `dataType` and returns it. Defaults to `true`. If additional attributes are queried along with `group` clauses, set to `false` to return all values of all returned rows. |
```

Comment:
Totally spaced on this one. I somehow managed to miss the inline documentation Will be fixed in the next update.